discord/voice_client.py

```diff
@@ -509,13 +494,14 @@ async def on_voice_server_update(self, data: VoiceServerUpdatePayload) -> None:
         self._voice_server_complete.set()
 
     async def voice_connect(self) -> None:
+        channel = await self.channel._get_channel() if self.channel else None
```

Comment:
Shouldn't coros be in the format `fetch`? For example, get_member, get_role aren't coros but fetch_member and fetch_role are, so shouldn't `channel._get_channel()` be `channel._fetch_channel()`?

Reply:
Internally everything is `get`. This change is just in case a `Messageable` makes its way here.